src/devicecontroller/DefaultDeviceController.ts

```diff
@@ -383,7 +383,7 @@ export default class DefaultDeviceController
 
       await this.chooseAudioTransformInputDevice(device);
     } else {
-      this.logger.info(`Choosing intrinsic input device ${device}`);
+      this.logger.info(`Choosing intrinsic input device ${JSON.stringify(device)}`);
```

Comment:
Why do we need to do this? device needs to be string anyway

Reply:
Maybe we should be clarify that to `Choosing intrinsic audio input device`?

Reply:
I added this initially because I saw the [Object object] being printed out here.